expandedTree/expandedTree.go

```diff
@@ -90,6 +94,7 @@ func GetBranch(node *ETree) []*ETree {
 	return branch
 }
 
+// GetBranch takes a root node and returns a slice of all leaf nodes in the tree.
```

Comment:
Copy paste error possibly

Reply:
Thank you for catching this.  I just pushed a fix